packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -691,6 +718,52 @@ export enum UpdateType {
   ROLLING_UPDATE = 'RollingUpdate',
 }
 
+/**
+ * AutoScalingGroup fleet change notifcations configurations.
+ * You can configure AutoScaling to send an SNS notification whenever your Auto Scaling group scales.
+ */
+export interface NotificationConfiguration {
+  /**
+   * SNS topic to send notifications about fleet scaling events
+   */
+  readonly notificationsTopic: sns.ITopic;
+
+  /**
+   * Which fleet scaling events triggers a notification
+   * @default all
+   */
+  readonly notificationTypes?: NotificationTypes;
+}
+
+/**
+ * Fleet scaling event notification type
+ */
+export enum NotificationType {
```

Comment:
consider calling this `FleetScalingEvents` or something similar to indicate that these are events